src/__tests__/commands/apps/oauth/generate.test.ts

```diff
@@ -0,0 +1,220 @@
+import type { objectDef, stringDef, updateFromUserInput } from '../../../../lib/item-input/index.js'
+import type { CommandArgs } from '../../../../commands/apps/oauth/generate.js'
+import { buildArgvMock, buildArgvMockStub } from '../../../test-lib/builder-mock.js'
+import { apiCommandMocks } from '../../../test-lib/api-command-mock.js'
+import { buildInputDefMock } from '../../../test-lib/input-type-mock.js'
+
+
+const {
+	apiCommandMock,
+	apiCommandBuilderMock,
+	itemInputHelpTextMock,
+} = apiCommandMocks('../../../..')
+
+const inputAndOutputItemMock =
+	jest.fn<typeof inputAndOutputItem<GenerateAppOAuthRequest, GenerateAppOAuthResponse>>()
+const inputAndOutputItemBuilderMock = jest.fn<typeof inputAndOutputItemBuilder>()
+jest.unstable_mockModule('../../../../lib/command/input-and-output-item.js', () => ({
+	inputAndOutputItem: inputAndOutputItemMock,
+	inputAndOutputItemBuilder: inputAndOutputItemBuilderMock,
+}))
+
+const inputProcessorMock = jest.fn<typeof inputProcessor>()
+jest.unstable_mockModule('../../../../lib/command/input-processor.js', () => ({
+	inputProcessor: inputProcessorMock,
+}))
+
+const chooseAppMock = jest.fn<typeof chooseApp>().mockResolvedValue('chosen-app-id')
+jest.unstable_mockModule('../../../../lib/command/util/apps-util.js', () => ({
+	chooseApp: chooseAppMock,
+}))
+
+const oauthAppScopeDefMock = buildInputDefMock<string>('Scopes Mock')
+jest.unstable_mockModule('../../../../lib/command/util/apps-util-input-primitives.js', () => ({
+	oauthAppScopeDef: oauthAppScopeDefMock,
+}))
+
+const updateFromUserInputMock = jest.fn<typeof updateFromUserInput>()
+const objectDefMock = jest.fn<typeof objectDef>()
+const stringDefMock = jest.fn<typeof stringDef>()
+jest.unstable_mockModule('../../../../lib/item-input/index.js', () => ({
+	objectDef: objectDefMock,
+	stringDef: stringDefMock,
+	updateFromUserInput: updateFromUserInputMock,
+}))
+
+
+const { default: cmd } = await import('../../../../commands/apps/oauth/generate.js')
+
+
+test('builder', () => {
+	const yargsMock = buildArgvMockStub<object>()
+	const {
+		yargsMock: apiCommandBuilderArgvMock,
+		positionalMock,
+		exampleMock,
+		epilogMock,
+		argvMock,
+	} = buildArgvMock<APICommandFlags, CommandArgs>()
+
+	apiCommandBuilderMock.mockReturnValueOnce(apiCommandBuilderArgvMock)
+	inputAndOutputItemBuilderMock.mockReturnValueOnce(argvMock)
+
+	const builder = cmd.builder as (yargs: Argv<object>) => Argv<CommandArgs>
+	expect(builder(yargsMock)).toBe(argvMock)
+
+	expect(apiCommandBuilderMock).toHaveBeenCalledExactlyOnceWith(yargsMock)
+	expect(inputAndOutputItemBuilderMock).toHaveBeenCalledExactlyOnceWith(apiCommandBuilderArgvMock)
+
+	expect(positionalMock).toHaveBeenCalledTimes(1)
+	expect(exampleMock).toHaveBeenCalledTimes(1)
+	expect(epilogMock).toHaveBeenCalledTimes(1)
+})
+
+describe('handler', () => {
+	const apiAppsGetOauthMock = jest.fn<typeof AppsEndpoint.prototype.getOauth>()
```

Comment:
Here and elsewhere, there's some inconsistency in how "OAuth" is capitalized when it appears in variable/object/type names.  Here it's treated as one word (`Oauth`) while in line 123, for example, it's treated as two (`OAuth`).